testsuite/integration/microprofile-health-tck/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If the plan is to add tcks for all MP specs, I think it would be tidier to group these under a top level folder, e.g testsuite/integration/microprofile-tck/health

Reply:
OK, I will ask about MP OT TCK plans and update the structure if @mjurc or @jstourac plans to have MP OT TCK in the TS